src/expr/impl/src/scalar/array_contain.rs

```diff
@@ -25,29 +25,35 @@ use risingwave_expr::function;
 ///
 /// ```slt
 /// query I
-/// select array[1,2,3] @> array[2,3];
+/// select array_contains(array[1,2,3], array[2,3]);
 /// ----
 /// t
 ///
 /// query I
-/// select array[1,2,3] @> array[3,4];
+/// select array_contains(array[1,2,3], array[3,4]);
 /// ----
 /// f
 ///
 /// query I
-/// select array[1,2,3] @> NULL;
+/// select array_contains(array[[[1,2],[3,4]],[[5,6],[7,8]]], array[2,3]);
+/// ----
+/// t
+///
+/// query I
+/// select array_contains(array[1,2,3], null::int[]);
```

Comment:
Nit: the explicit cast on `null` is unnecessary. The original query is supposed to work without change.